src/main/java/com/google/devtools/build/lib/analysis/test/TestConfiguration.java

```diff
@@ -286,6 +286,14 @@
         help = "If true, then Bazel will run coverage postprocessing for test in a new spawn.")
     public boolean splitCoveragePostProcessing;
 
+    @Option(
+        name = "zip_undeclared_test_outputs",
```

Comment:
FYI, we usually start with `experimental_` for a new feature flag and set its default value to `false` if it is a boolean value. But in this case, calling `zip_undeclared_test_outputs` and defaulting to `true` make more sense.

Reply:
Acknowledged.